File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/FormBox/Features/ArgumentsPropertyForm.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;

using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;

#endregion

namespace Gizmox.WebGUI.Forms.CompanionKit.Controls.FormBox.Features
{
    public partial class ArgumentsPropertyForm : Form
    {
        public ArgumentsPropertyForm()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Handles Load event of Form
        /// </summary>
        private void ArgumentsPropertyForm_Load(object sender, EventArgs e)
        {
            // Check if argument with key needed exists and 
            // set its value to apropriate Label as a text

            if (Context.Arguments["Color"] != null)
                lblColor.Text = string.Format("Color: {0}", Context.Arguments["Color"]);

            if (Context.Arguments["Size"] != null)
                lblSize.Text = string.Format("Size: {0}", Context.Arguments["Size"]);
        }
    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/FormBox/Features/ArgumentsPropertyForm.vb
Imports Gizmox.WebGUI.Forms
Imports Gizmox.WebGUI.Common

Namespace Gizmox.WebGUI.Forms.CompanionKit.Controls.FormBox.Features

    Public Class ArgumentsPropertyForm
        Public Sub New()

            ' This call is required by the Visual WebGui UserControl Designer.
            InitializeComponent()

        End Sub

        Private Sub ArgumentsPropertyForm_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
            ' Check if argument with key needed exists and 
            ' set its value to apropriate Label as a text

            If Not Context.Arguments("Color") Is Nothing Then
                lblColor.Text = String.Format("Color: {0}", Context.Arguments("Color"))
            End If

            If Not Context.Arguments("Size") Is Nothing Then
                lblSize.Text = String.Format("Size: {0}", Context.Arguments("Size"))
            End If

        End Sub

    End Class

End Namespace
